deploy/kubernetes/releases/csi-digitalocean-dev/snapshot-validation-webhook.yaml

```diff
@@ -20,9 +20,9 @@ webhooks:
   - name: "validation-webhook.snapshot.storage.k8s.io"
```

Comment:
Since v8.0.0, snapshot-validation-webhook doesn't validate anything but volumesnapshotclasses. New example configuration here: https://github.com/kubernetes-csi/external-snapshotter/blob/v8.0.1/deploy/kubernetes/webhook-example/admission-configuration-template
And here's the changelog entry: https://github.com/kubernetes-csi/external-snapshotter/blob/master/CHANGELOG/CHANGELOG-8.0.md#urgent-upgrade-notes